src/launcher/features/settings/AddArtifactoryTokenDialog.tsx

```diff
@@ -52,10 +52,14 @@ export default () => {
             onCancel={hideDialog}
         >
             <p>
-                Enter an Artifactory token to use for retrieving apps. You can
-                get an identity token from{' '}
+                Enter an identity token to use for retrieving apps. To get a
+                token, go to{' '}
                 <ExternalLink href="https://files.nordicsemi.com/ui/user_profile" />
-                .
+                , log in and generate an identity token there.
+            </p>
+            <p>
+                As Nordic employee, you should be able to log in, otherwise ask
+                your Nordic contact for an account if you are eligible.
```

Comment:
```suggestion
                As Nordic employee, you are able to log in by default. If not a Nordic employee, ask
                your Nordic contact for an account.
```
We should [sic] avoid "should". Also, "if you are eligible" puts the verification onus on the user, which the user might not be able to know until contacting Nordic, so I find this part confusing.

Reply:
I like leaving out “should”. And I think we can also leave out the “by default”. I will do this and other changes in a manual commit.

Reply:
@greg-fer Did the changes (and similar ones in other places). 

“If not a Nordic employee, ask your Nordic contact for an account.” makes it sound like we give everyone an account and access who asks for it. I wanted to avoid this with “if you are eligible”, but I am fine with both, so I removed it.